pages/checkout.page.ts

```diff
@@ -1,14 +1,92 @@
 import { Page, Locator, expect } from "@playwright/test";
+import { CreditCardPaymentFragment } from "./fragments/payment.fragment";
 
 export class CheckoutPage {
   readonly path: string = "/checkout";
+  readonly cartStep: CartStep = new CartStep(this.page);
```

Comment:
It is very nice that you created separate components for each step. Let's move them to separate files. Good practice is - one class one file